config/locales/en.yml

```diff
@@ -185,9 +185,13 @@ en:
     new_references: References
     new_references_start: Choose a referee
     new_references_name: What is the referee’s name?
+    new_references_edit_name: Edit the name of the referee
```

Comment:
Not part of this PR but these keys are not really leveraging YAML nesting very well.

Reply:
I'll card up